pages/knowyourrights/index.tsx

```diff
@@ -199,15 +199,23 @@ const KnowYourRights = () => {
                   <HighlightOffIcon className={classes.icons} />
                 </button>
                 {activeItem && (
-                  <img
+                  <a
                     style={{ marginTop: '2rem' }}
-                    id="simple-modal-title"
-                    src={
+                    href={
                       language === ENGLISH
                         ? `${activeItem.flyerPDF}`
                         : `${activeItem.spanishFlyerPDF} `
                     }
-                  />
+                    download="Flyer"
+                  >
+                    <img
+                      src={
```

Comment:
![Screen Shot 2021-03-22 at 3 20 03 PM](https://user-images.githubusercontent.com/72828520/112065575-17d64780-8b22-11eb-964f-17e89b303039.png)

The text and pictures narrowing isn't optimal. also, these link to png/jpg files from an outside source? i think it makes more sense to host them locally, especially.